themes/classic/_dev/css/components/search-widget.scss

```diff
@@ -34,20 +34,82 @@
   }
 }
 
+#close-search,
+.search-cross {
+  display: none;
+}
+#close-search:checked ~ ul.ui-autocomplete.ui-front.ui-widget-content {
+  display: none;
+}
+
+ul {
+  &.ui-autocomplete.ui-front {
+    &.ui-widget-content {
+      border: none;
+      border-radius: 0;
+      .ui-menu-item {
+        &:hover {
+          border: none;
+          border-radius: 0;
+        }
+        a {
+          padding-top: .47rem;
+          padding-left: .813rem;
+          padding-bottom: .47rem;
+          &:hover {
+            border: none;
+            border-radius: 0;
+            background: $gray-light-third !important;
```

Comment:
I read your answer to the same question from @PierreRambaud 
I can see there is no additional class you can target, but then the proper solution would be to update the js that initialize the autocomplete and force additional classes that you can use afterwards

Besides it makes me think that this fix probably shouldn't be in the core code but rather in the module code, then you can update the javascript and css inside the module

Besides, when I look at the design attached to the issue I can see there are some thumbnails integrated in the autocomplete list, they are not present in the current `_renderItem` function from the module so you won't be able to display them

Also the category name is missing from the `SearchController` response, this should be fixed in the core this time, but since the new design doesn't display them it probably won't be necessary